chainstate/test-suite/src/tests/pos_accounting_reorg.rs

```diff
@@ -138,18 +155,33 @@ fn stake_pool_reorg(#[case] seed: Seed) {
         // where reorg never happened.
         //
         // Construct fresh `genesis -> b -> c` chain as a reference
-        let expected_accounting_data = {
+        let expected_storage = {
             let storage = Store::new_empty().unwrap();
             let mut tf = TestFramework::builder(&mut rng)
                 .with_storage(storage.clone())
                 .with_chainstate_config(tf.chainstate().get_chainstate_config())
+                .with_chain_config(chain_config)
                 .build();
 
             {
                 // manually add block_a info
                 let mut db_tx = storage.transaction_rw().unwrap();
                 db_tx.set_block_index(&block_a_index).unwrap();
                 db_tx.add_block(&block_a).unwrap();
+
+                // reorg leaves a trace in delta index, because deltas are never removed on undo;
```

Comment:
I hope these are reused and not left for nothing...

Reply:
good question. Actually, it's only reused if that block ever gets to the main chain again